tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -13,7 +13,7 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
 
 1. In the TiDB Cloud console, get the TiDB Cloud account ID and external ID of the target TiDB cluster.
 
-    1. In the TiDB Cloud console, choose your target project, and navigate to the **Clusters** page.
+    1. In the TiDB Cloud console, navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
 
     2. Locate your target cluster, click **...** in the upper-right corner of the cluster area, and select **Import Data**. The **Data Import** page is displayed.
 
```

Comment:
L16-18, L22, L106-108: Currently, user can import data using three ways:

- On the **Clusters** page, click **...**>**Import Data**
- On the Cluster Overview page, click **Import Data**
- Navigate to the **Import** tab (on the left navigation) of a cluster.

L24&L112: Keep it consistent with UI **Guide for getting the required Role ARN**
<img width="904" alt="image" src="https://user-images.githubusercontent.com/60599231/207005757-33222a29-5f0c-41e3-b447-dd5ee749d205.png">